### PR TITLE
25 create best stadiums section

### PR DESCRIPTION
**Pull request by ahmedirheem**
related #25

### CODE REVIEW

**Review by Saeed99Madi — CHANGES_REQUESTED** [review]
Great Job Ahmed, Just needs to use the styled function that supported from MUI,

**Review by Emanalabsi — APPROVED** [review]

File: client/src/components/best-stadiums/BestStadiums.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    image:
+      'https://www.verizon.com/about/sites/default/files/2022-05/stadium-1230x690.jpg',
+    name: 'ملعب الاتحاد',
+    rate: 4.5,
+    address: 'غزة - الرمال - شارع الوحدة',
+    description: 'ملعب ملعب ملعب ملعب ملعب ملعب ملعب ملعب ملعب ملعب ',
+  },
+];
+
+const BestStadiums: FC = () => {
```
Comment (alaaahmade) [inline client/src/components/best-stadiums/BestStadiums.tsx:40]: done forget add type of returning function : ReactElement

Comment (alaaahmade) [inline client/src/components/best-stadiums/BestStadiums.tsx:40]: done forget to add the type of returning function : ReactElement

Comment (alaaahmade) [inline client/src/components/best-stadiums/BestStadiums.tsx:40]: done forget add type of returning function : ReactElement

File: client/src/components/best-stadiums/Stadium.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+interface Props {
+  image: string;
+  name: string;
+  rate: number;
+  address: string;
+  description: string;
+}
+
+const Stadium: FC<Props> = ({ image, name, rate, address, description }) => {
```
Comment (alaaahmade) [inline client/src/components/best-stadiums/Stadium.tsx:21]: done forget to add the type of returning function : ReactElement

**Review by alaaahmade — CHANGES_REQUESTED** [review]

**Review by alaaahmade — APPROVED** [review]
great job dear Ahmed

**Comment by Heba-Adwan** [comment]
solve conflict

**Review by Heba-Adwan — APPROVED** [review]

**Review by Saeed99Madi — APPROVED** [review]

**Pull request merged**